torch/_inductor/compile_fx.py

```diff
@@ -1383,6 +1632,172 @@ def _send_to_child(
         return self._run_in_child(pickled_input)
 
 
+class _OutOfProcessFxCompile(_SerializedFxCompile):
+    """
+    Represents an FxCompile which is run outside the current process (in
+    either a subprocess or possibly even a separate machine).
+    """
+
+    def _postprocess(self, output: _WireProtocolOutput) -> None:
+        # Since our metrics were gathered in a subprocess make sure to add them
+        # here.
+        CachedMetricsHelper.apply_deltas(output.metrics)
```

Comment:
Which specific metrics do we expect to apply here? Does it include counters? CompilationMetrics?

Reply:
I added the metrics that seemed to be checked by tests. We certainly could/should add any metrics needed in the future.

Reply:
There's tests in test_structured_trace for structured logs, fwiw, once we get there